daras_ai_v2/language_model.py

```
@@ -307,6 +310,36 @@ def run_language_model(
         return ret
 
 
+def local_image_urls_to_base64(
+    messages: list[ConversationEntry],
+) -> list[ConversationEntry]:
+    ret = []
+    for message in messages:
+        content = []
+        if img_urls := get_entry_images(message):
+            img_bytes = [requests.get(img_url).content for img_url in img_urls]
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)